src/app/app.component.html

```diff
@@ -2,11 +2,5 @@
     <lib-editor [editorConfig]="editorConfig" (editorEmitter)="editorEventListener($event)"></lib-editor>
 </ng-container>
 <div *ngIf="!editor" class="ui container d-flex flex-jc-center flex-ai-center p-20">
-    <button class="sb-btn sb-btn-normal sb-btn-outline-primary mr-10 font-weight-bold fs-1" (click)="setType('question')" >Question Editor</button>
```

Comment:
Disable these buttons rather than removing

Reply:
I have updated the code. Please check the same